R/utils-fetch.R

```diff
@@ -1,6 +1,16 @@
 # This R file's purpose is for extracting values from json 
 # The functions shouldn't modify the data, and should handle missing json or empty json parameters
 
+#' Fetch Field
+#'
+#' @description Generalize fetch function that fetches the provided field
+#' @param reportObect The full report data loaded from the report JSON
+#' @param field The field to fetch from the data
+fetchReportField <- function(reportObect, field){
```

Comment:
This is a function meant to fetch any supplied JSON field from the report, not a typo of fieldVists. I'll change the name to something else to make that more clear though since it does look like a typo as it is.